### ISSUE TITLE
Non-members and decision making

### ISSUE THREAD

**Issue by rhiaro** [OpenDataServices/decision-pipeline#5]
"Employee" is currently [defined as](https://decision-pipeline.readthedocs.io/en/latest/glossary.html#term-Employee):

> A non-member employee of the co-op does not directly participate in making decisions [...]

I think there is scope for guidance about decisions that _can_ be actively participated in by non-members. Non-members needn't be excluded from _all_ decisions. For example, if we were deciding the frequency and format of whole-coop meetings over the next year, this is a decision that all employees can participate in making, without it affecting Director-level responsibilities.

(Or is the Decision Pipeline strictly _only_ for Director-level type decisions?)

**Comment by ScatteredInk** [comment]
> (Or is the Decision Pipeline strictly only for Director-level type decisions?)

Yes, at this stage. In part because I don't think we want to add in any ambiguity/optionality around responsibilities - it should just be about directing members' attention where it needs to go, for now, and ensuring that decisions progress as needed. 

That said, I think the glossary gives the wrong impression around non-member involvement. "Does not directly participate" contradicts other parts of the documentation around "Stakeholders", and some undocumented assumptions around delegated decisions - these will often rely on the participation of employees. I think the distinction we want to make is probably between members and non-members, and participation in "final decision-making processes".

> I think there is scope for guidance about decisions that can be actively participated in by non-members.

Yes, agree, although I suspect it sits above the decision pipeline itself - or will eventually lead to us breaking out _all_ the guidance into a separate thing.
